testing/run_api_tests.sh

```diff
@@ -91,5 +122,9 @@ if [[ $MIGRATION -eq 1 ]]; then
     kill -TERM $pid
 fi
 
+if [ "$docker_down" -eq 1 ]; then
+    docker compose -f "$script_dir/docker-compose.yml" down
+fi
+
 echo "Exiting with status $exit_code"
 exit $exit_code
```

Comment:
During one test, I still encountered that issue where the testing Wakapi instance was still running in the background and blocking the port when there had been a failure during the tests. Couldn't really reproduce that again, though. So maybe quickly revise the "killing" logic again, if you can find a clue on how that happened, but otherwise I think it's fine :+1:.

Reply:
Revised. Should be somewhat more robust.